src/api/routes/auth/forgot.ts

```diff
@@ -12,7 +12,18 @@ const router = Router();
 
 router.post(
 	"/",
-	route({ body: "ForgotPasswordSchema" }),
+	route({
+		requestBody: "ForgotPasswordSchema",
+		responses: {
+			204: {},
+			400: {
+				body: "APIErrorResponse",
+			},
```

Comment:
APIErrorResponse doesn't contain keys for captcha which uses 400. Is it reasonable to have the response bodys be an array, ie `["APIErrorResponse", "CaptchaErrorResponse"]`?